microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -149,50 +267,134 @@ To use the **Network Console**, complete the following steps.
+## Focus Mode
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## DevTools Tooltips
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Detached Elements
+
+Memory leaks in web applications are a difficult to locate and repair.
+
+The new Detached Elements panel in Microsoft Edge DevTools helps investigate and resolve a common source of memory leaks.
+
+Memory leaks occur when the JavaScript code of the application retains an increasing number of objects in memory. The JavaScript code should release these objects so that the memory space can be reused by the browser.
+
+<!-- ???more needed, probably with an image, Detached Elements tutorial is in progress??? -->
+
+
+<!-- ====================================================================== -->
+## VS Code themes for the DevTools
+
+Placeholder.
+
+
 <!-- ====================================================================== -->
 ## Open source files in Visual Studio Code
 <!-- in Experiments as of 96, at end of main list -->
 
-The **Open source files in Visual Studio Code** experiment replaces the code editor of the Sources tool with Visual Studio Code, for editing local files.  When you turn on this experiment, Developer Tools detects when you edit a local file, and prompts you to select a folder to use as your Workspace.
+The **Open source files in Visual Studio Code** experiment replaces the code editor of the Sources tool with Visual Studio Code, for editing local files. When you turn on this experiment, Developer Tools detects when you edit a local file, and prompts you to select a folder to use as your Workspace.
 
-After you select a folder to use as your Workspace, selecting any link to a file in DevTools opens the file in Visual Studio Code, rather than in the code editor of the Sources tool in DevTools.
+When you select a folder to use as your Workspace, selecting any link to a file in DevTools opens the file in Visual Studio Code. This action previously opened the file in the code editor of the Sources tool in DevTools.
 
 :::image type="content" source="../media/experiment-sources-in-code-editor-open.msft.png" alt-text="Selecting a file link in the Styles tool opens the file in Visual Studio Code" lightbox="../media/experiment-sources-in-code-editor-open.msft.png":::
 
-Any edits that you make in DevTools will now change the file on the hard drive and sync live with Visual Studio Code.  You can read about setting up your workspace in [Opening source files in Visual Studio Code](../sources/opening-sources-in-vscode.md).
+Any edits that you make in DevTools now change the file on the hard drive and sync live with Visual Studio Code. You can read about setting up your workspace in [Opening source files in Visual Studio Code](../sources/opening-sources-in-vscode.md).
 
 This experiment is available starting with Microsoft Edge version 96.
 
 
```

Comment:
Currently (97 & 98) there's a bug, a dup checkbox that starts with "Microsoft Edge ": might want to include here for tracking/completeness: 
Microsoft Edge automatically pretty print in the Sources Panel

Reply:
I am leaving it off ... if it is here in the next version I will worry about it then.